lib/rails_core_extensions/active_record_cloning.rb

```diff
@@ -27,6 +26,26 @@ def clones_attributes_except(*attributes)
       cloned_attributes_hash[:exclude] = attributes.map(&:to_sym)
     end
 
+    def clones_attributes_reset
+      @cloned_attributes = nil
+    end
+
+    def exclude_attributes(cloned, excludes)
+      excluded_attributes(excludes).each do |attr|
+        cloned.send("#{attr}=", nil)
+      end
+    end
+
+    def excluded_attributes(excludes)
+      all_attributes = attribute_names.map(&:to_sym)
+      included_attributes = if attributes_included_in_cloning.empty?
```

Comment:
Could be nice to have this as a private method (yes, I can see it's only moved code 😛 )